wellregistry/registry/models.py

```diff
@@ -29,16 +133,17 @@ class Registry(models.Model):
     We could refactor later.
 
     """
-    # these will become lookups with                    foreign keys
-    agency_cd = models.CharField(max_length=20)       # AGENCY_LOV.AGENCY_CD
-    well_depth_units = models.IntegerField()          # UNITS_DIM.UNIT_ID
-    alt_datum_cd = models.CharField(max_length=10)    # ALT_DATUM_DIM.ADATUM_CD
-    alt_units = models.IntegerField()                 # UNITS_DIM.UNIT_ID
-    horz_datum = models.CharField(max_length=10)      # HORZ_DATUM_DIM.HDATUM_CD
-    nat_aquifer_cd = models.CharField(max_length=10)  # NAT_AQFR.NAT_AQFR_CD
-    country_cd = models.ForeignKey(CountryLookup, on_delete=models.CASCADE)       # COUNTRY.COUNTRY_CD
-    state_cd = models.CharField(max_length=2)         # STATE.STATE_CD and STATE.COUNTRY_CD
-    county_cd = models.CharField(max_length=3)        # COUNTY.COUNTY_CD and COUNTY.STATE_CD and COUNTY.COUNTRY_CD
+    # these columns use foreign keys
+    agency_cd = models.ForeignKey(AgencyLovLookup, on_delete=models.PROTECT) # AGENCY_LOV.AGENCY_CD
```

Comment:
I think it might make sense to drop the cd from these fields. Any thoughts?

Reply:
Probably makes sense. In the WQP  agency table, the field is called code. We could stick with that convention.

Reply:
Just to be clear, I was recommended changing agency_cd to agency. I think it would be fine to change the Lookup models as you describe too.

Reply:
Missed that these columns are in the registry table. Definitively should be named, _cd dropped.